setup.py

```diff
@@ -35,7 +35,7 @@
     packages=find_packages(exclude=["tests*", "examples*", "tutorials*"]),
     install_requires=[
         "numpy>=1.7",
-        "scipy>=1.0.0",
+        "scipy>=1.0.0,<=1.4.1",
```

Comment:
I can't remember if we discussed why the upper bound on scipy was placed here

Reply:
my bad. I'll revert this. no need for this.